pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf

```diff
@@ -1607,10 +1607,10 @@
     "summary_complete": "Thank you for providing contact information for your physician(s) and hospital(s). You can update the information you provided by clicking on the \"Update\" button.",
 
     "intro": {
-      "p1": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Thank you very much for your consent to have your child participate in a research study called Project Singular.\", \"Thank you very much for your consent to participate in our research study, Project Singular.\")",
+      "p1": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Thank you very much for your consent to have your child participate in our research study, Project Singular.\", \"Thank you very much for your consent to participate in our research study, Project Singular.\")",
       "p2": "To proceed with our study, we are asking you to release $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") relevant medical records to our staff. First, you must sign this medical release form so that our study staff may obtain copies of $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") medical records, securely store them, and use them for this study. ",
       "p3": "Relevant medical records include operation notes, last cardiology clinic note, echo reports, and catheterization reports.",
-      "p4": "For our staff to receive your records, you have several options. Please select the option that you would like to use:",
+      "p4": "For our staff to receive $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") records, you have several options. Please select the option that you would like to use:",
 
```

Comment:
Missed a few of these "your child" vs "your" in the text. I hope I got them all now.